netbox/extras/migrations/0051_migrate_customfields.py

```diff
@@ -67,7 +67,7 @@ def migrate_customfieldvalues(apps, schema_editor):
         cf_data = model.objects.filter(pk=cfv.obj_id).values('custom_field_data').first()
         try:
             cf_data['custom_field_data'][cfv.field.name] = deserialize_value(cfv.field, cfv.serialized_value)
-        except ValueError as e:
+        except Exception as e:
```

Comment:
Is it possible to narrow in on the two exceptions that cause a issue only?

For example (ValueError,DoesNotExist) as e?

Reply:
Yes, but I don't see the point since the exception is immediately re-raised anyway.

For circumstances we haven't thought about yet, IMO it's better to get the backtrace *with* an indication of the data that triggered the error, than a backtrace without the data.

Reply:
Ah, I was half asleep and didn't notice the second raise.

This wouldn't silently continue then though.  Was that the point?

Reply:
Correct.  I did it this way (a) for minimum change, and (b) to be consistent with the existing policy, which is to abort on errors, rather than silently discard bad data.  See also #5575, which is in a different function (`migrate_customfield_defaults`) and also causes migration to fail.

If I'd had this error reported back originally, then I could have cleansed the data manually before migrating.